specs/language/statements.tex

```diff
@@ -1,2 +1,74 @@
 \Ch{Statements}{Stmt}
 \Sec{Label Statements}{Stmt.Label}
+\Sec{Attributes}{Stmt.Attr}
+\Sub{Unroll Attribute}{Stmt.Attr.Unroll}
+The Attribute \textbf{[unroll]} and \textbf{[unroll(n)]} attribute qualifiers can be used to specify that a loop (\textbf{for}, \textbf{while} and \textbf{do while} loops) can be unrolled. This attribute qualifier can be used to specify full unrolling or partial unrolling by a specified amount. This is a compiler hint and the compiler may ignore this directive.
+
+n is the loop unrolling factor and must be a positive integral compile time constant expression. If n is not specified, the compiler determines the unrolling factor for the loop. This attribute is not compatible with the [loop] attribute.
```

Comment:
Should we do something to typographically highlight that `n` is special?  I see other parts of the spec use `\texttt{n}`

Reply:
n italisized vs regular:
![image](https://github.com/microsoft/hlsl-specs/assets/1802579/97dda70e-959c-43a6-a707-1ab0c746a2d4)
Its such a suttle difference maybe we should bold or underline instead?

Reply:
I went ahead with italics assuming it just looks weird on my end.

Reply:
I'm mostly calling out the differences in style I see between this new part of the spec and other parts.  @llvm-beanz may have more thoughts on how we should be tackling this in the doc.